### DISCUSSION TITLE
Thief (Neutral)

### DISCUSSION THREAD

**Discussion by Alf21** [Alf21/TTT2#752] [Role Ideas]
### Thief

**Basic description**

The Thief was born without a purpose… he has, however, been gifted the power to claim one!

**Role abilities and Gameplay**

The Thief spawns as a neutral role and, just like the Jester, without the ability to harm anyone. He does, however, have a special tool: The Role Stealer.

The Role Stealer takes the form of a knife with one use. When attacked, the victim doesn’t receive any immediate damage or notification. After thirty seconds, however, the Thief will have taken over the victim’s role while the victim is left behind roleless and, just like the Thief once was, without the ability to deal damage.

The twist: The Thief can’t damage the player he has stolen the role from. The victim, however, can still damage the Thief. If the victim manages to find the one who took his role, they can kill him and claim it back for themselves.

If not, the victim is doomed to lose the round, no matter what, because they only get their role back if the Thief is killed by them and not by anyone else.

As a victim, life is pretty hard, since you can be killed by anyone without repercussions. It’s better not to be suspected as a Thief’s victim. 

_Optional:_ The victim could also simply be invincible. This could be toggleable for higher player counts where it’d be harder for them to find the Thief.

_Optional: The victim could also get their role back when the Thief dies in general. This removes some pressure from the victim since they don’t necessarily have to be the one to track and kill the Thief. This could be toggleable, depending on which you prefer._

**Win conditions**

Since the Thief steals roles, his win condition depends on the role he has taken over.

**Why it might be interesting**

It creates a fun little game, where the Thief’s victim desperately needs to identify the Thief (preferably by shooting people without being detected, since only the Thief will be damaged) to reclaim their role.

The Thief, meanwhile, plays his victim’s role, with the added difficulty of avoiding them since he can’t damage them. Stealing someone’s role allows the Thief to live and win the round – if he can avoid his victim’s revenge!

If the Thief dies with a stolen role, his victim will be left behind with no role and no means to win. If a Thief senses that he’s about to die at someone else’s hands, he might reconsider and leave his victim to kill him, giving them another shot at winning…?

**Interactions with other roles**

If the Thief steals the Jester role, his victim can kill him and reclaim their role without having the Thief win as the Jester.

The Thief can’t damage his victim, but if his stolen role happens 

Idea by: @Skatcat

---

Answer by @LunexSan: Give the thief's victim a radar of some sort to find the thief, otherwise this role might suggest the victim to just randomly shoot at people until he finds the thief.

**Comment by mexikoedi** [comment]
There is a similar role but not exactly how it is described here: https://steamcommunity.com/sharedfiles/filedetails/?id=2652144870
